actionview/lib/action_view/helpers/asset_tag_helper.rb

```diff
@@ -364,6 +375,16 @@ def multiple_sources_tag_builder(type, sources)
           end
         end
 
+        def resolve_image_source(source, skip_pipeline)
+          if source.is_a?(Symbol) || source.is_a?(String)
```

Comment:
I'm not sure the "can't resolve" part is necessary there, given an existing reasonable exception from the delegation. But even so, apart from the word "image", that all seems like reasonable behaviour for `polymorphic_url`.